src/bin/test_bam_decompress.rs

```diff
@@ -0,0 +1,48 @@
+extern crate rust_htslib;
```

Comment:
This test is not needed - it's essentially testing that writing sam output to "-" works, i.e. `sam_write1` works (which belong in the underlying library). Also, there isn't an obvious way to redirect output of stdout temporarily over on the `C` side, so it will need to be tested with a subprocess. This is complicated by `cargo test` capturing stdout.  But since this test doesn't belong here anyway, I will remove it.